app/styles/main.scss

```diff
@@ -151,19 +151,33 @@ footer {
       border-bottom: 0;
     }
     tr.open > td:first-child {
-      background: url('../images/plus.png') no-repeat 0px 5px;
-      background-size: 20px 20px;
+      padding-left: 0px;
+    }
+    tr.open > td:first-child:before {
+      content: "";
+      float: left;
+      width: 45px;
+      height: 20px;
+      background: url('../images/accordian-arrow.png') no-repeat 0 -5px;
+      background-size: 30px;
+    }
+    tr.close > td:first-child:before {
+      content: "";
+      float: left;
+      width: 45px;
+      height: 20px;
+      background: url('../images/accordian-arrow.png') no-repeat 0 -35px;
+      background-size: 30px;
```

Comment:
Yes. The above is sick-in-mouth horrible CSS, hence the comment on the commit, but it was the only way I could make use of the stupid 2-icons-in-1-stupid-png nonsense that is the GOV UK frontend toolkit.